python/nano/notebooks/pytorch/cifar10/nano-trainer-example.ipynb

```diff
@@ -224,7 +224,8 @@
     "BATCH_SIZE = 32\n",
     "NUM_WORKERS = 0\n",
     "data_module = prepare_data(PATH_DATASETS, BATCH_SIZE, NUM_WORKERS)\n",
-    "EPOCHS = int(os.environ.get('FIT_EPOCHS', 30))"
+    "MAX_STEPS = int(os.environ.get('MAX_STEPS', -1))\n",
+    "EPOCHS = 30"
```

Comment:
let's make sure it will finish at least one epoch.

Reply:
Done.
Unit test will run 3 epochs.